### PR TITLE
Push ensembles separarate from responses to new storage

### PR DESCRIPTION
**Pull request by ManInFez**
**Approach**
_Short description of the approach_

### CODE REVIEW

**Comment by ManInFez** [comment]
first step of #1133

File: ert_shared/models/iterated_ensemble_smoother.py [hunk 1]
```diff
@@ -95,9 +95,14 @@ def runSimulations(self, arguments):
             analysis_success = current_iter > pre_analysis_iter_num
             if analysis_success:
                 analysis_module_name = self.ert().analysisConfig().activeModuleName()
-                previous_ensemble_name = dump_to_new_storage(reference=None if previous_ensemble_name is None else (previous_ensemble_name, analysis_module_name))
+                # Push ensemble, parameters, observations to new storage
+                analysis_module_name = self.ert().analysisConfig().activeModuleName()
```
Comment (oysteoh) [inline ert_shared/models/iterated_ensemble_smoother.py:99]: duplicated line #97

**Comment by ManInFez** [comment]
Test this please!

**Review by oysteoh — CHANGES_REQUESTED** [review]
The duplicated line should be fixed, other than that you are good to go from my point of view! 🚀

File: ert_shared/models/iterated_ensemble_smoother.py (hunk 1 above)
Comment (ManInFez) [reply ert_shared/models/iterated_ensemble_smoother.py:98]: Done!

**Review by oysteoh — APPROVED** [review]

**Pull request merged**